app/helpers/duration.spec.js

```diff
@@ -0,0 +1,20 @@
+const assert = require('assert')
+
+const duration = require('./duration')
+
+describe('duration', function () {
+    it('should return miliseconds duration', function () {
+      const miliSecondsDuration = 999
+      assert.equal(miliSecondsDuration + ' ms', duration(miliSecondsDuration))
+    })
+
+    it('should return seconds duration', function () {
+        const secondsDuration = 59000
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/UFABCNextOps/ufabc-next-server/pullRequest?prid=8443142)
```suggestion
    const secondsDuration = 59000
```